packages/office-ui-fabric-react/src/components/Image/Image.Props.ts

```diff
@@ -22,6 +22,11 @@ export interface IImageProps extends React.HTMLProps<HTMLImageElement>  {
    * Image source to display if an error occurs loading the image indicated by src.
    */
   errorSrc?: string;
+
+  /**
+   * If true, the image frame will expand to fill its parent container.
+   */
+  shouldMaximizeFrame?: boolean;
```

Comment:
I think that this should really be another enum. So I would add stretch to the ImageFit enum. Additionally there is already some css that does height: 100% and width: 100% in 
```
.ms-Image-image--scaleWidthHeight {
  height: 100%;
  width: 100%;
}
```

So you might be able to leverage that or rename it to stretch. I'm not sure about renaming it though.